### PR TITLE
Add `sublabel` to `DetailRow`

### PR DESCRIPTION
**Pull request by xplato**

### CODE REVIEW

File: src/lib/ui/layout/DetailRow.tsx [hunk 1]
```diff
@@ -2,15 +2,22 @@ import type { PropsWithChildren } from 'react'
...
   label,
+  sublabel,
   children,
 }: PropsWithChildren<DetailRowProps>): JSX.Element {
   return (
     <div className='seam-detail-row'>
-      <p className='seam-row-label'>{label}</p>
+      <div className='seam-detail-row-label-wrap'>
+        <p className='seam-row-label'>{label}</p>
+        {sublabel != undefined && (
```
Comment (xplato) [inline src/lib/ui/layout/DetailRow.tsx:17]: Using `null` here will make the element show up in the DOM, messing up the layout of rows that don't have this property. This is also a flaw in the update to `DetailSection` with the optional `label` prop—the element is still in the DOM.

File: src/lib/ui/layout/DetailRow.tsx [hunk 2]
```diff
@@ -2,15 +2,22 @@ import type { PropsWithChildren } from 'react'
...
   label,
+  sublabel,
   children,
 }: PropsWithChildren<DetailRowProps>): JSX.Element {
   return (
     <div className='seam-detail-row'>
-      <p className='seam-row-label'>{label}</p>
+      <div className='seam-detail-row-label-wrap'>
+        <p className='seam-row-label'>{label}</p>
+        {sublabel !== undefined && (
```
Comment (razor-x) [inline src/lib/ui/layout/DetailRow.tsx:17]:
```suggestion
        {sublabel != null  && (
```

Comment (xplato) [reply src/lib/ui/layout/DetailRow.tsx:17]: This is what I mean, yeah. If we use `null` here, the element will still render in the DOM

Comment (xplato) [reply src/lib/ui/layout/DetailRow.tsx:17]: (Same with the recent change to `DetailSection` with the optional `label` prop)

Comment (razor-x) [reply src/lib/ui/layout/DetailRow.tsx:17]: Should be treated the same via `==`

image

Comment (xplato) [reply src/lib/ui/layout/DetailRow.tsx:17]: See this:

https://github.com/seamapi/react/assets/87919558/f6f79acc-1620-405a-b29c-b63262a1cd2a

Comment (xplato) [reply src/lib/ui/layout/DetailRow.tsx:17]: Sorry it's because I have `!==` not `!=`—glossed over that, my bad

**Pull request merged**